custom-recipes/ab-testing-split/recipe.py

```diff
@@ -0,0 +1,69 @@
+import dataiku
+from dataiku.customrecipe import get_input_names_for_role, get_output_names_for_role, get_recipe_config
+
+from design.ab_split import check_sample_size, split_into_groups
+from design.split_format import add_group_column, group_to_df
+
+# ==============================================================================
+# SETUP
+# ==============================================================================
+input_name = get_input_names_for_role('user_list')[0]
+input_dataset = dataiku.Dataset(input_name)
+input_df = input_dataset.get_dataframe()
+
+folder_name = get_input_names_for_role('folder')[0]
+folder = dataiku.Folder(folder_name)
+
+groups_name = get_output_names_for_role('groups')[0]
+groups_dataset = dataiku.Dataset(groups_name)
+
+A_output = get_output_names_for_role("A_group")
+B_output = get_output_names_for_role("B_group")
+
+if A_output:
+    A_output_name = A_output[0]
+    A_group_dataset = dataiku.Dataset(A_output_name)
+
+if B_output:
+    B_output_name = B_output[0]
+    B_group_dataset = dataiku.Dataset(B_output_name)
+
+config = get_recipe_config()
+reference_column = config.get("user_reference")
+
+size_definition = config.get("sample_size_definition")
+if size_definition == "web_app":
+    parameter_filename = config.get("parameters")
+    tracking = folder.read_json(parameter_filename)
+    n_A = int(tracking["n_A"])
```

Comment:
`n_A`, `n_B` are not really a great naming.

Reply:
replaced by size_A, size_B in https://github.com/dataiku/dss-plugin-ab-testing/pull/1/commits/a2c91e1042972154cfc9799db57d51536fb5463c